server/core/src/https/views/admin/groups.rs

```diff
@@ -207,14 +209,26 @@ pub(crate) async fn edit_group(
         Some(ScimValueKanidm::String(query.account_name)),
     );
 
+    let (group_info, _) =
+        get_group_info(group_uuid, state.clone(), &kopid, client_auth_info.clone()).await?;
+
+    // query.description can't be Some("") since axum deserializes "" to None.
+    // Also meaning that I can't check if someone wants to unset a field or couldn't set the field.
+    // Thus, I check if there's a difference below to make up for this.
```

Comment:
I'd say removing the description if it's set to an empty string seems sensible?

Reply:
yes that is the goal, the issue related to access is explained in the comment